nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/security/impl/ACLImpl.java

```diff
@@ -56,6 +57,13 @@
     private final boolean isReadOnly;
 
     public ACLImpl(String name, boolean isReadOnly) {
+        if (name == null) {
+            log.debug("ACL name is null");
+            if (log.isTraceEnabled()) {
+                log.trace(new NuxeoException("ACL name is null"));
```

Comment:
`log.trace` and others will just log a message, and not the full stacktrace, if there's only one parameter. You need something like:

    log.trace("ACL name is null", new NuxeoException("trace"));